tests/tests/swfs/audio/aac/NOTE.txt

```diff
@@ -0,0 +1,7 @@
+ 
+ ffmpeg -f lavfi -i "sine=frequency=444:duration=1:sample_rate=44100" -af "volume=5" -ac 2 -c:a aac tone_stereo_44100hz.flv
+ ffmpeg -f lavfi -i "sine=frequency=444:duration=1:sample_rate=48000" -af "volume=5" -ac 2 -c:a aac tone_stereo_48000hz.flv
+ ffmpeg -f lavfi -i "sine=frequency=444:duration=1:sample_rate=22050" -af "volume=5" -ac 2 -c:a aac tone_stereo_22050hz.flv
+ ffmpeg -f lavfi -i "sine=frequency=444:duration=1:sample_rate=44100" -af "volume=5" -ac 1 -c:a aac tone_mono_44100hz.flv
+ ffmpeg -f lavfi -i "sine=frequency=444:duration=1:sample_rate=48000" -af "volume=5" -ac 1 -c:a aac tone_mono_48000hz.flv
+ ffmpeg -f lavfi -i "sine=frequency=444:duration=1:sample_rate=22050" -af "volume=5" -ac 1 -c:a aac tone_mono_22050hz.flv
```

Comment:
Renamed it  `regenerate_sounds.sh` (as they are already checked in).
Do you think it's fine without a shebang?

Reply:
Huh, you made me wonder how the shell/kernel behaves without a shebang (the file has its executable bit set). According to my research it should fine, especially that we have very simple commands here. (I would personally add a shebang anyway :D)

Reply:
The good thing if I don't add a shebang is that it has the _theoretical potential_ to work within any (reasonable) shell under the Sun, even on different platforms (like CMD.exe, or PowerShell, or whatever).

Reply:
That's true even with a shebang, because a shebang is just a comment (for any reasonable shell) :D Adding a (ba)sh shebang kinda makes sense considering the file is named .sh

But of course any decision here is okay for me.